scripts/mkstdlibs.py

```diff
@@ -4,7 +4,16 @@
 
 URL = "https://docs.python.org/{}/objects.inv"
 PATH = "isort/stdlibs/py{}.py"
-VERSIONS = [("2", "7"), ("3", "6"), ("3", "7"), ("3", "8"), ("3", "9"), ("3", "10"), ("3", "11")]
+VERSIONS = [
+    ("2", "7"),
+    ("3", "6"),
+    ("3", "7"),
+    ("3", "8"),
+    ("3", "9"),
+    ("3", "10"),
+    ("3", "11"),
+    ("3", "12"),
+]
```

Comment:
```suggestion
VERSIONS = [
    ("3", "8"),
    ("3", "9"),
    ("3", "10"),
    ("3", "11"),
    ("3", "12"),
]
```

Let's also remove unsupported versions.